concepticondata/conceptrelations.tsv

```diff
@@ -770,3 +770,12 @@ SOURCE	SOURCE_GLOSS	RELATION	TARGET	TARGET_GLOSS
 3805	SHY OR ASHAMED	narrower	1894	ASHAMED
 3809	SCREAM (PRODUCE A CRY)	producedby	2522	SCREAM (NOISE)
 298	SHRIEK	similar	3809	SCREAM (PRODUCE A CRY)
+3213	HIRE OR RENT	narrower	3831	RENT (SOMETHING)
+947	STREAM	narrower	3821	BROOK
```

Comment:
STREAM is already an instance of 3311	FLOWING BODY OF WATER. Wouldn't this also be the relation for BROOK?

Reply:
The difference between RIVER, STREAM and BROOK seems fuzzy anyway... But I guess a RIVER is defined by not flowing into another FLOWING BODY OF WATER, and a BROOK is a small STREAM. Thereby, a BROOK is a kind of STREAM.